org.kde.ghostwriter.yml

```diff
@@ -0,0 +1,95 @@
+app-id: org.kde.ghostwriter
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: ghostwriter
+rename-icon: ghostwriter
+finish-args:
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--share=ipc"
+  - "--share=network"
+  - "--device=dri"
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - name: cmark
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/commonmark/cmark.git
+        tag: 0.31.0
+        commit: 8fbf029685482827828b5858444157052f1b0a5f
+        x-checker-data:
+          type: anitya
+          project-id: 9159
+          stable-only: true
+          tag-template: $version
+  - name: cmark-gfm
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/github/cmark.git
+        tag: 0.29.0.gfm.13
+        commit: 587a12bb54d95ac37241377e6ddc93ea0e45439b
+        x-checker-data:
+          type: anitya
+          project-id: 17699
+          stable-only: true
+          tag-template: $version
+  - name: MultiMarkdown
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/fletcher/MultiMarkdown-6.git
+        tag: 6.7.0
+        commit: 6b6155d5041a023d0de44adf182506c52202a03c
+        x-checker-data:
+          type: anitya
+          project-id: 131019
+          stable-only: true
+          tag-template: $version
+  - name: Pandoc
+    buildsystem: simple
```

Comment:
Why is this not being built from source?

Reply:
pandoc has release tarball with binary

Reply:
also its complex to build